client/service/apis.go

```diff
@@ -198,8 +198,8 @@ func (c *DatabricksClient) configureHTTPCLient() {
 	}
 }
 
-// IsAzure returns true if Azure is configured
-func (c *DatabricksClient) IsAzure() bool {
+// UsingAzureAuth returns true if Azure authentication is configured, this is not a valid check to determine if we are running on Azure
+func (c *DatabricksClient) UsingAzureAuth() bool {
 	return c.AzureAuth.resourceID() != ""
```

Comment:
i really wonder how this changes things?..

Reply:
For what this method is currently used for (the tests), this check would work fine. Renaming it would avoid that anyone uses it for anything else. What would you propose?